flight/search/cell_map.py

```diff
@@ -0,0 +1,140 @@
+"""
+Summary
+-------
+Defines the CellMap class and has some basic tests at the bottom of the file.
+"""
+
+from cell import Cell
+from helper import get_bounds
```

Comment:
Local imports should be relative to root directory
```suggestion
from flight.search.cell import Cell
from flight.search.helper import get_bounds
```